src/constants/chains.ts

```diff
@@ -21,6 +21,8 @@ export enum SupportedChainId {
   CELO_ALFAJORES = 44787,
 
   BNB = 56,
+
+  BASE = 8453,
```

Comment:
at some point our team might remove this enum entirely, like i started to do in #615 but for now this works